### PR TITLE
Feature: 14.04 removal

### PR DESCRIPTION
**Pull request by lsh-0**
* removes 18.04 as an alternate build
* removes the last of 14.04 state handling

### CODE REVIEW

File: salt/search/config/etc-default-gearman-job-server [hunk 1]
```diff
@@ -1,10 +0,0 @@
-# This is a configuration file for /etc/init.d/gearman-job-server; it allows
-# you to perform common modifications to the behavior of the gearman-job-server
-# daemon startup without editing the init script (and thus getting prompted by
-# dpkg on upgrades).  We all love dpkg prompts.
-
-# elife 2019-05-28: this is a Ubuntu-only shim for upstart, Gearman doesn't support config from a file
-
-# Parameters to pass to gearmand.
-PARAMS="--libpq-conninfo=postgres://{{ pillar.search.gearman.db.username }}:{{ pillar.search.gearman.db.password }}@127.0.0.1/{{ pillar.search.gearman.db.name }} --queue-type=Postgres --verbose=INFO"
```
Comment (giorgiosironi) [inline salt/search/config/etc-default-gearman-job-server]: this is now in https://github.com/elifesciences/search-formula/blob/363d8f8aad3bcc983a237ee3a7b033c87b09fa82/salt/search/config/lib-systemd-system-gearman-job-server.service#L14, for reference

File: salt/search/gearman-persistence.sls [hunk 2]
```diff
@@ -25,28 +25,12 @@ gearman-db:
...
-        - source: salt://search/config/etc-default-gearman-job-server
-        - template: jinja
-        - require:
-            - gearman-daemon # elife.gearman-server.sls
-            - gearman-db
 
 gearman-service:
-    {% if osrelease == '14.04' %}
-    cmd.run:
-        # I do not trust anymore Upstart to see changes to init scripts when using `restart` alone
```
Comment (giorgiosironi) [inline salt/search/gearman-persistence.sls]: I guess we trust systemd instead? Assuming there's a `systemctl daemon-reload` somewhere

Comment (lsh-0) [reply salt/search/gearman-persistence.sls]: I haven't had a problem with daemon-reload + salt for ages now, I suspect it's doing it automatically behind the scenes. I'll keep an eye out for it in future though.

File: salt/search/config/etc-default-gearman-job-server (hunk 1 above)
Comment (lsh-0) [reply salt/search/config/etc-default-gearman-job-server]: yes, it took me a while to unravel this one. This /etc/default/gearman-job-server stuff was sourced by the init.d script that was in turn used by Upstart - none of it official/endorsed by gearman, just the guy who packaged it for Ubuntu. As you say, it's now all contained in the systemd `.service` file

**Pull request merged**